src/refract/Registry.cc

```diff
@@ -41,10 +40,6 @@ std::string Registry::getElementId(IElement& element)
         throw LogicError("Element has no ID");
     }
 
-    // FIXME: remove dependecy on SosSerializeCompactVisitor
-    SosSerializeCompactVisitor v;
-    VisitBy(*it->second, v);
-
```

Comment:
PR looks good code wise, but I couldn't understand why this part is removed. Would appreciate a small reply explaining. Especially, why we were visiting in the first place if it's not needed.

Reply:
We didn't. This was dead code.